docs/current_docs/manuals/user/cloud/572923-get-started.mdx

```diff
@@ -97,7 +93,7 @@ The general procedure to connect Dagger Cloud with a CI provider/CI tool is:
     :::
   - Add the secret to your CI environment as a variable named `DAGGER_CLOUD_TOKEN`.
 - For **Team** plan subscribers with ephemeral CI runners only: Add a wait step (example below) for the Docker configuration to allow the Dagger Engine to push cache volumes to the experimental Dagger Cloud distributed cache. Subscribers on the **Individual** plan do not have access to the experimental distributed cache and do not need to add this step.
-- If you are using GitHub Actions, install the Dagger Cloud GitHub app for GitHub Checks. This app adds a GitHub Check to your pull requests and links to the run status and pipeline visualization for the commit in Dagger Cloud.
+- If you are using GitHub Actions, install the Dagger Cloud GitHub app for GitHub Checks. This app adds a GitHub Check to your pull requests for your Dagger pipeline run and links to the pipeline visualization for the commit in Dagger Cloud.
```

Comment:
I found this ambiguous so here is an attempt to make it clearer. I may have got it wrong though.

```suggestion
- If you are using GitHub Actions, install the Dagger Cloud GitHub app for GitHub Checks. This app adds a GitHub check to your pull requests. Each check links to the corresponding pipeline visualization in Dagger Cloud.
```

Reply:
The app has a new UX, which prints the command out. For `dagger call`, you get the status of the run reported back to the check, which is a big improvement. I'm waiting for clarification from @aluzzardi about whether this is a NEW app or an update to the existing one.